.github/workflows/cypress.yml

```diff
@@ -102,7 +102,17 @@ jobs:
         run: npx @wordpress/env@latest start
 
       - name: Run Cypress Tests
-        run: npm run test:e2e
+        uses: replayio/action-cypress@v0.2.2
+        with:
+          api-key: ${{ secrets.RECORD_REPLAY_API_KEY }}
+          issue-number: ${{ github.event.pull_request.number }}
+          browser: 'Replay Chromium'
+          command: npm run test:e2e --
+          upload-all: true
+
+      - run: npx @replayio/replay ls --json
+        shell: bash
+        name: List Replays
```

Comment:
This isn't necessary but can be useful for debugging. I'd suggest adding `if: ${{ always() }}` to ensure that it runs even if a test fails.